test/pk-src/copy-src.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ ! -d riscv-pk ]; then
+  rsync --exclude ".git" -r ../../riscv-pk .
+  patch riscv-pk/bbl/bbl.c < test.patch
```

Comment:
I wish there was a cleaner testing method than maintaining out-of-tree patches that are potentially brittle.
To lessen the problem, `copy-src.sh` should begin with `set -e`, so that marshal would terminate at an earlier point if the patch failed to apply.  (Same goes for the linux-src test.)

Reply:
Actually, I guess that would be redundant since the exit code would be inherited from the patch command being the last one in the script, but `set -e` is a good precaution anyway.

Reply:
'set -e' is a good idea. I added that. 

I'm open to suggestions for a more reliable way to test a non-standard bbl, but this way should be pretty stable (it's a one-line change that doesn't depend on much). Anyway, it will only stop working if we bump riscv-pk, and we would run this test if we did that anyway.